test/language/expressions/generators/params-trailing-comma-length.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (C) 2016 Jeff Morrison. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+description: Check that trailing commas are permitted in function argument lists
```

Comment:
Ok. I've updated the generator descriptions to include "generator".

What level of clarity are you looking for for other descriptions though? The current descriptions seem pretty clear to me, but happy to add additional context where necessary.